de_project/de_app/urls.py

```diff
@@ -4,4 +4,5 @@
 
 urlpatterns = [
     path('', views.index, name='index'),
```

Comment:
I would just remove this view if it isn't going to be used

Reply:
See previous comment about the views. This might be useful for testing.